timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -254,6 +138,17 @@ def save(self) -> None:
 
         self.models = models
 
+    @classmethod
+    def load(cls, path: str, reset_paths: bool = False) -> Self:
+        load_path = os.path.join(path, cls.trainer_file_name)
+        if not reset_paths:
+            return load_pkl.load(path=load_path)
+        else:
+            obj = load_pkl.load(path=load_path)
+            obj.set_contexts(path)
+            obj.reset_paths = reset_paths
+            return obj
```

Comment:
This looks equivalent to `AbstractTabularTrainer.load()`. Should we move the method to `AbstractTrainer`?

Reply:
thanks!